rust_snuba/rust_arroyo/src/backends/kafka/config.rs

```diff
@@ -85,6 +90,10 @@ mod tests {
                 "queued.max.messages.kbytes".to_string(),
                 "1000000".to_string(),
             )])),
+            Some(HashMap::from([(
+                "max.poll.interval.ms".to_string(),
+                "1000000".to_string(),
```

Comment:
What's this for?

Reply:
It's a unit test. Refactored it to allow passing in key value pairs to the kafka config from the resolver.